internal/configs/sharding/mainnet.go

```diff
@@ -11,6 +12,10 @@ const (
 	blocksPerShard     = 16384  // 2^14
 	mainnetV1Epoch     = 1
 	mainnetV2Epoch     = 5
+
+	mainnetMaxTxAmountLimit               = 1e3 // unit is in One
+	mainnetMaxNumRecentTxsPerAccountLimit = 10
+	mainnetMaxTxsPerBlockLimit            = 8000
```

Comment:
This is a very high number. It will cause a 2mb block, which is huge for our network. Better Just cap it at 1000 at most.